Assets/Plugins/LeapMotion/Experimental/HierarchyRecording/Examples/Cannon/Recordings/Cannon 01.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: e1217cb2e871afd44b1891247d1299eb
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
```

Comment:
It looks like you ran some of the hierarchy recorder scenes (which created some recordings), and the meta files got committed.  This causes warnings about having to delete these meta files on client machines.  Could you please delete these .meta files from the PR?